projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -2736,6 +2736,8 @@
     "ElementwiseUnaryIntModule_basic",
     "ElementwiseFloatTensorGtIntTensorModule_basic",
     "MaskedFillTensorFloatValueModule_basic",
+    "MultinomialModule_basic",
+    "MultinomialModule2D_basic",
```

Comment:
Shouldn't these tests be passing, since you've added the Linalg lowering too?

Reply:
@vivekkhandelwal1 This confuses me too, this doesn't happen in any other backend but in ONNX this throws an error `RuntimeError: invalid multinomial distribution (sum of probabilities <= 0)`, which I believe is coming from the [CPU kernel](https://github.com/pytorch/pytorch/blob/eea4ece256d74c6f25c1f4eab37b3f2f4aeefd4d/aten/src/ATen/native/cpu/MultinomialKernel.cpp#L183) for multinomial. 

My implementation works in the other backends, and it seems that this error occurs when computing a reference value using pytorch so I would think it is an issue with tracing or with my onnx to torch lowering, however, in my onnx to torch lowering I'm simply just passing the input tensor through to torch.

Do you have any idea why this could be happening?

Reply:
This might be a tracing issue. We can go ahead with this PR for now, since you've added it as a Xfail.